robber/expect.py

```diff
@@ -60,9 +60,14 @@ def registered(cls, name):
         else:
             return True
 
+    @property
+    def not_to(self):
+        self.is_negated = True
+        return self
+
     def __setup_chaining(self):
-        self.to   = self
-        self.be   = self
-        self.a    = self
-        self.an   = self
+        self.to = self
+        self.be = self
+        self.a = self
+        self.an = self
         self.have = self
```

Comment:
@anhhuy1605 Not yet, I think. I will think about it again when I consider about the double (triple...) negated matcher.